lib/jsdom/browser/domtohtml.js

```diff
@@ -8,7 +8,7 @@ exports.domToHtml = function (iterable) {
   let ret = "";
   for (const node of iterable) {
     if (node.nodeType === NODE_TYPE.DOCUMENT_NODE) {
-      ret += parse5.serialize(node, {
+      ret += parse5.serialize(idlUtils.tryWrapperForImpl(node), {
```

Comment:
Can you explain why this is necessary in the commit message? What changed, or were we buggy before?